src/x509.cc

```diff
@@ -45,6 +45,85 @@ std::string parse_args(const Nan::FunctionCallbackInfo<v8::Value>& info) {
   return *String::Utf8Value(info[0]->ToString());
 }
 
+
+
+NAN_METHOD(verify) {
+  Nan::HandleScope scope;
+  OpenSSL_add_all_algorithms();
```

Comment:
Not sure that pulling in another module is the best solution, especially when we're not using anything in the crypto module other than it loading the algorithms. Calling `OpenSSL_add_all_algorithms` in our code won't cause problems if someone uses `require('crypto')`, as subsequent calls to `OpenSSL_add_all_algorithms` will work perfectly fine.